docs/_posts/2022-04-19-power-bladesworn.md

```diff
@@ -0,0 +1,63 @@
+title: Power Bladesworn
+tags: power warrior bladesworn
+tagline: "'It is my duty to keep the people of Cantha safe.'<br/>-- Minister Li"
+spec: bladesworn
+---
+
+This Bladesworn Warrior build averages at about 13 APM (Tier 1). This is a melee-heavy build, making movement control a potential factor.
+
+## Tier Notes
+
+### Rotation Tier 1
+1. keep auto-attacking until your [![](https://wiki.guildwars2.com/images/thumb/c/c4/Heat.png/20px-Heat.png) flow bar](https://wiki.guildwars2.com/wiki/Flow) is full
+2. press [![Dragon Trigger](https://wiki.guildwars2.com/images/thumb/b/b1/Dragon_Trigger.png/33px-Dragon_Trigger.png) `F2`](https://wiki.guildwars2.com/wiki/Dragon_Trigger) and wait until your ![](https://wiki.guildwars2.com/images/thumb/9/98/Gunsaber_Cartridge_%28uncharged%29.png/12px-Gunsaber_Cartridge_%28uncharged%29.png)charges are full
+3. press [![Dragon Slash—Force](https://wiki.guildwars2.com/images/thumb/b/b5/Dragon_Slash%E2%80%94Force.png/33px-Dragon_Slash%E2%80%94Force.png) `1`](https://wiki.guildwars2.com/wiki/Dragon_Slash%E2%80%94Force) and wait for your [![Sheathe Gunsaber](https://wiki.guildwars2.com/images/thumb/8/87/Sheathe_Gunsaber.png/33px-Sheathe_Gunsaber.png) `F1`](https://wiki.guildwars2.com/wiki/Sheathe_Gunsaber) to become ready
+4. press [![Sheathe Gunsaber](https://wiki.guildwars2.com/images/thumb/8/87/Sheathe_Gunsaber.png/33px-Sheathe_Gunsaber.png) `F1`](https://wiki.guildwars2.com/wiki/Sheathe_Gunsaber) and wait for your [![](https://wiki.guildwars2.com/images/thumb/c/c4/Heat.png/20px-Heat.png) flow bar](https://wiki.guildwars2.com/wiki/Flow) again
+
+### Rotation Tier 2
+Same as Tier 1 + use [![Whirling Axe](https://wiki.guildwars2.com/images/thumb/e/ec/Whirling_Axe.png/33px-Whirling_Axe.png) Axe 5](https://wiki.guildwars2.com/wiki/Whirling_Axe) off cooldown. **Make sure to wait until all hits are done**
+
+## Traits
+
+Template code:
+
+`[&DQIEHzMWRCtwAAAAqQCpAK8AqgA9AawAnADuAAAAAAAAAAAAAAAAAAAAAAA=]`
+
+---
+
+<div
+  data-armory-embed='skills'
+  data-armory-ids='14389,14404,14410,14479,14355'
+>
+</div>
+<div
+  data-armory-embed='specializations'
+  data-armory-ids='4,51,68'
+  data-armory-4-traits='1444,1449,1437'
+  data-armory-51-traits='1413,1484,1369'
+  data-armory-68-traits='2225,2302,2239'
+>
+</div>
+<script async src='https://unpkg.com/armory-embeds@^0.x.x/armory-embeds.js'></script>
+
+## Gearing
+
+- Full Berserker armor & trinkets
+- [![](https://wiki.guildwars2.com/images/thumb/9/96/Superior_Rune_of_the_Thief.png/27px-Superior_Rune_of_the_Thief.png) Runes of the Thief](https://wiki.guildwars2.com/wiki/Superior_Rune_of_the_Thief)
+- Berserker Axe/Axe Force/Accuracy
+
+## Variants
+
+Berserker can be replaced by Marauder and Knight, though knight will perform considerably worse.
+
+Keep your sigils the same on this one, as crit is very important and force is pretty strong.
+
+## Other References
+
+Check out [my video discussing showcasing](https://www.youtube.com/watch?v=AaPuiW0ulGM) this build.
```

Comment:
"discussing and showcasing"?

Reply:
"discussing" was a usual copy & paste :/